src/app/fdctl/run/tiles/fd_dedup.c

```diff
@@ -31,6 +35,7 @@ typedef struct {
   ulong * tcache_ring;
   ulong * tcache_map;
 
+  ulong             num_in_idx_to_parse;
```

Comment:
How about `unparsed_in_cnt` With a comment that the first `unparsed_in_cnt` in links do not have the parsed `fd_txn_t`